### DISCUSSION TITLE
Fresh install of Directus with existing database

### DISCUSSION THREAD

**Discussion by Chris2Brooks** [directus/directus#2783] [Archive]
Hey folks,

So after some awesome playing with my server I pretty much broke it to a point where I figured starting fresh would be better. 
Don't you just hate it?! 😏 

Anyway's database still works great, so i want to do a SQL dump of my db at it to a new hosted MySQL instance at AWS and host my Directus on AWS Lightsail. 

I know Directus supports existing databases. What do I need to do to get my project back to where it was? (I still have access to Directus' files and folders on the server)

- Just connect my database that has the SQL dump restored?
- Add the config files of the project?

Appreciate any help. I'm copnsidering doing a blog post too on this process so it's easier for future kids breaking servers and projects as they go.

**Comment by rijkvanzanten** [comment]
> Just connect my database that has the SQL dump restored?
> Add the config files of the project?

Yes and yes, if you have a full backup of the previous directus installation, there's no other steps you have to take to use it

**Reply by Chris2Brooks** [reply]
Nice! Thanks Rijk 🙌
